src/transformers/models/bit/configuration_bit.py

```diff
@@ -49,8 +49,14 @@ class BitConfig(PretrainedConfig):
         hidden_act (`str`, *optional*, defaults to `"relu"`):
             The non-linear activation function in each block. If string, `"gelu"`, `"relu"`, `"selu"` and `"gelu_new"`
             are supported.
+        global_padding (`str`, *optional*, defaults to `None`):
+            padding strategy to use for most of the layers, it can be either `"valid"`, `"same"`, or `None`.
```

Comment:
```suggestion
        global_padding (`str`, *optional*):
            Padding strategy to use for the convolutional layers. Can be either `"valid"`, `"same"`, or `None`.
```
"most of the layers" is a bit vague, isn't this for all convolutional layers?

Reply:
I put that because we use this flag on the embedding layer too

Reply:
The embedding layer is also convolutional right?

Reply:
Yes great point!